### ISSUE TITLE
Center the image

### ISSUE THREAD

**Issue by Naheel-Azawy** [Kharacternyk/pacwall#26]
The picture is a bit off from the center... Not sure if that's normal or wrong but it annoys me a lot for some reason... Is it possible to automatically center it?

![image](https://user-images.githubusercontent.com/5764614/78396330-dd655900-75f7-11ea-88e1-1078e07e610c.png)

Awesome work btw!
And yes I know I have tons of packages and I need serious cleanups

**Comment by Kharacternyk** [comment]
Btw, that barely visible blue dots are potential orphans. I should consider to use a somewhat brighter color.

**Comment by Naheel-Azawy** [comment]
@Kharacternyk probably because of the black background. I guess it looks fine with the gray background. My packages are pure mess...

**Comment by Kharacternyk** [comment]
I added `-c` option in 01fe701. It allows to change the package that will be put into the center of the circles. It tends to change the appearance of the graph dramatically. Maybe you could try to guess a package that will work for you?

**Comment by Naheel-Azawy** [comment]
That's nice thanks! I tried with glibc... Here's what I got...
![image](https://user-images.githubusercontent.com/5764614/78505424-13334a80-777c-11ea-834d-75a803bbabec.png)

**Comment by Kharacternyk** [comment]
Yeah, some guesswork is required and it isn't even guaranteed to work.

I'm still looking for a proper solution.

**Comment by Z2h-A6n** [comment]
Hi, made a pull request that centers the image when a root package is specified with `-c ROOT`. See #44 . It relies on having `twopi` also make a plain-text representation of the graph, and `grep`-ing for the root node in that text. I don't know how to do something like this if `-c` is not used, since it's not clear how to get the name of the root node in that case.

**Comment by Kharacternyk** [comment]
@Z2h-A6n fixed this in v1.5 but I let the issue hang around because v2.* doesn't have a fix.
